### PR TITLE
Fix: selecting a disabled language (e.g germany) as a local language 

### PR DESCRIPTION
**Pull request by syphax-bouazzouni**
fix https://github.com/biodivportal/ontoportal_web_ui/issues/9 and https://github.com/biodivportal/ontoportal_web_ui/issues/10

### CODE REVIEW

**Comment by Bilelkihal** [comment]
@syphax-bouazzouni 
I prefer keeping the red color for errors,
but the language fix seems good, what will happen if the local language is disabled?

File: app/assets/stylesheets/ontologies.scss [hunk 1]
```diff
@@ -2,6 +2,10 @@ $ont-metadata-bg-color: #e2ebf0;
 $widget-table-border-color: #EFEFEF;
 
 
+.resolvability-check svg path{
```
Comment (Bilelkihal) [inline app/assets/stylesheets/ontologies.scss:5]: If we merge the PR this need to be deleted

Comment (syphax-bouazzouni) [reply app/assets/stylesheets/ontologies.scss:5]: this was also required by https://github.com/biodivportal/ontoportal_web_ui/issues/9

Comment (Bilelkihal) [reply app/assets/stylesheets/ontologies.scss:5]: Yes in our case we don't want it

Comment (syphax-bouazzouni) [reply app/assets/stylesheets/ontologies.scss]: OK dine in https://github.com/ontoportal-lirmm/bioportal_web_ui/pull/748/commits/af478b44b4efbe3b4bc32ebd146af744143b8766

Comment (Bilelkihal) [reply app/assets/stylesheets/ontologies.scss]: Perfect, I'll merge it then

**Pull request merged**